generators/entity-client/templates/react/src/main/webapp/app/entities/entity-dialog.tsx.ejs

```diff
@@ -31,7 +31,17 @@ _%>
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { Modal, ModalHeader, ModalBody, ModalFooter, Button, Label<% if (haveFieldWithJavadoc) { %>, UncontrolledTooltip<% } %> } from 'reactstrap';
-import { AvForm, AvGroup, AvInput, AvField } from 'availity-reactstrap-validation';
+<%_ 
+  isLocalDate = false;
+  for (idx in fields) {
+    fieldType = fields[idx].fieldType;
+    if (fieldType === 'LocalDate') {
+      isLocalDate = true;
+      break;
+    }
+  }
+_%>
+import { AvForm, AvGroup, AvInput<%_ if (isLocalDate) { _%>, AvField <%_ } _%> } from 'availity-reactstrap-validation';
```

Comment:
The `<%_` & `_%>` ejs syantx is used to slurp new lines so they should only be used in new lines so that rendered code will not have those, for inline usage like this use `<% %> syntax